vslib/src/tests.cpp

```diff
@@ -777,6 +778,19 @@ void test_get_stats()
 
     ASSERT_TRUE(values[0] == 127);
     ASSERT_TRUE(values[1] == 77);
+
+    std::vector<sai_object_id_t> supported_obj_list;
```

Comment:
please make this separate method, you added your case in test_get_stats, and your changes are not related to stats